CONTRIBUTING.rst

```diff
@@ -25,42 +25,205 @@ Pull Request Checklist
    guidelines <https://github.com/deepchecks/deepchecks/blob/master/CONTRIBUTING.rst>`__
 -  Check if your changes are consistent with the
    `guidelines <#general-guidelines-for-contribution>`__
--  Changes are consistent with the `coding
-   style <#coding-style>`__
--  Run the `unit
-   tests <#running-unit-tests>`__
+-  Refer to the `Windows users <#windows-users>`__ section if your development environment
+   is Windows.
+
+   - `Creating Virtual Environment <#creating-virtual-environment>`__
+   - `Installing Dependencies <#installing-dependencies>`__
+   - `Running Unit Tests <#windows-running-unit-tests>`__
+   - `Coding Style <#windows-coding-style>`__
+
+-  Refer to the `Linux/Mac users <#linux-and-mac-users>`__ section if your development environment
```

Comment:
Put this section above windows, as it's more common IMO.

Reply:
Done